docs/content/how-do-i-write-policies.md

```diff
@@ -1091,19 +1039,20 @@ It becomes clear that this is incorrect when you use the [`some`](#some-keyword)
 keyword, because the rule is true whenever there is SOME app that is not a
 bitcoin-miner:
 
-```ruby
+```live:eg/incorrect_no_bitcoin_some:module
 no_bitcoin_miners {
     some i
-    app := apps[i]
+    app := data.apps[i]
```

Comment:
Many examples in this doc refer to the data defined at the end of the page. It's assumed those definitions are in-scope. This should be `apps` not `data.apps`.

Reply:
Reverted this change in a couple of places you pointed out where I did it unnecessarily, in this case though I think it was to be done this way given that the query's use `with` to test different values. I can replace `with` with extra module blocks that define `apps` for each query if you want.

Reply:
To clarify, otherwise, `var apps is unsafe`. Better option: add a hidden module block that defines it.

Reply:
Adding a hidden module that includes the definitions from the end of the page is preferable.